plugin/cli.go

```diff
@@ -33,44 +36,57 @@ func getSignedJwt(role string, m map[string]string) (string, error) {
 		serviceAccount = credentials.ClientEmail
 	}
 	if serviceAccount == "" {
+		// Check if the metadata server is available.
+		if metadata.OnGCE() {
```

Comment:
This is _super_ nitty, but it could be a bit cleaner if you negated this (saves on some indentation). For example:

```go
if !metadata.OnGCE() {
  return "", errors.New("could not obtain...")
}

// Metadata server is available, pull creds from there
metadataClient := metadata.NewClient(&http.Client{})
...
```

Reply:
Fixed